elements/rh-code-block/docs/50-accessibility.md

```diff
@@ -0,0 +1,6 @@
+{% include 'accessibility/ariaguide.md' %}
+{% include 'accessibility/wcag.md' %}
+{% include 'accessibility/2.1.1-A.md' %}
+{% include 'accessibility/2.1.3-AAA.md' %}
+{% include 'accessibility/2.4.3-A.md' %}
+{% include 'accessibility/2.5.5-AAA.md' %}
```

Comment:
I recommend only including the accessibility sections that are applicable to the element here.

Reply:
these are the ones i found in the xd

could you send a commit to fix this page?